spec/emittr/callback_spec.rb

```diff
@@ -21,32 +21,34 @@
   end
 
   describe '#==' do
-    context 'when same callbacks' do
+    context 'when same callback is passed' do
       it 'returns true' do
         cb = Emittr::Callback.new &callback
         expect(cb == callback).to be true
       end
     end
 
-    context 'when different callback but wrapper is same' do
-      it 'returns true' do
-        diff_callback = Proc.new {}
-        cb = Emittr::Callback.new &callback
-        cb.wrapper = diff_callback
+    context 'when different callback is passed' do
+      context 'and wrapper is also different' do
+        it 'returns false' do
+          wrapper_callback = Proc.new {}
+          unset_callback = Proc.new {}
 
-        expect(cb == diff_callback).to be true
-      end
-    end
+          cb = Emittr::Callback.new &callback
+          cb.wrapper = wrapper_callback
 
-    context 'when a different callback is passed' do
-      it 'returns false' do
-        wrapper_callback = Proc.new {}
-        unset_callback = Proc.new {}
+          expect(cb == unset_callback).to be false
+        end
+      end
 
-        cb = Emittr::Callback.new &callback
-        cb.wrapper = wrapper_callback
+      context 'and wrapper is the same' do
```

Comment:
nop, I put `and` just to connect the messages on the `contexts`. do you prefer with it?